arbeitszeit_flask/database/db.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from sqlalchemy import Engine, create_engine
+from sqlalchemy.orm import DeclarativeBase, scoped_session, sessionmaker
+
+
+class Database:
```

Comment:
Do you mean "we would **NOT** need to deal with global variables"?

I do not understand why it is a problem to have a global variable in this context and why your proposel gives us more flexibility?

The Flask docu seems to recommend global variables and not using the g object when using SQLAlchemy?


 If you are wondering why we don’t have to care about threads here (like we did in the SQLite3 example above with the g object): that’s because SQLAlchemy does that for us already with the scoped_session.
 (https://flask.palletsprojects.com/en/stable/patterns/sqlalchemy/#declarative)